assets/javascript/apps/pipeline/nodes/NodeOutputs.tsx

```diff
@@ -11,16 +11,25 @@ export default function NodeOutputs({data}: {
   const generateOutputHandle = (outputIndex: number) => {
     return multipleOutputs ? `output_${outputIndex}` : "output";
   };
+
   const generateOutputLabel = (outputIndex: number, output_label:string) => {
-    if (multipleOutputs && outputIndex === 0) {
+    // Get the default index from params, defaulting to 0 if not defined
+    const defaultIndex = data.params.defaultKeywordIndex !== undefined ? data.params.defaultKeywordIndex : 0;
```

Comment:
Nitty comment: I'm wondering if we should do this, since there is a default value specified in the node params, so the default index will always be specified. Thoughts?